packages/database/supabase/migrations/20250619021503_correct_migrations.sql

```diff
@@ -0,0 +1,69 @@
+-- complete migration 20250610201801_create_access_table.sql
+
+revoke delete on table "public"."access_token" from "anon";
+
+revoke references on table "public"."access_token" from "anon";
+
+revoke trigger on table "public"."access_token" from "anon";
+
+revoke truncate on table "public"."access_token" from "anon";
+
+revoke update on table "public"."access_token" from "anon";
+
+-- rename in schema was not ported to migration 20250611180757_concept_upsert_functions.sql
+
+alter function "public".local_concept_to_db_concept(data concept_local_input) rename to _local_concept_to_db_concept;
```

Comment:
What error is this fixing?  Could you link the the source of the error?

Reply:
Before and after work on a migration, `supabase db diff` (with supabase in stopped state) should say there are no differences between applying the schema directly, or through the sequence of migrations. 
In this case, the content of schema and migration have different permissions; so the `db diff` provided these changes to align the migrations with the schema. It's entirely possible that the migration was correct and the schema was wrong; in which case we should change the schema and not the migration.

Reply:
Right.  What I'm saying is that we should like to where this happened so that someone inspecting this PR has a breadcrumb and as a way to ensure these changes actually reflect remote schema.

EG: https://github.com/DiscourseGraphs/discourse-graph/pull/215/files#diff-22bac1fb1de78f55daaeedaaa408e1bd563554544a152bf0c82630d8675a69daR139